executor/executor_test.go

```diff
@@ -990,12 +990,12 @@ func (s *testSuite) TestUnion(c *C) {
 
 	_, err := tk.Exec("select 1 from (select a from t limit 1 union all select a from t limit 1) tmp")
 	c.Assert(err, NotNil)
-	terr := errors.Trace(err).(*errors.Err).Cause().(*terror.Error)
+	terr := errors.Cause(errors.Trace(err)).(*terror.Error)
```

Comment:
can be simplified to "errors.Cause(err).(*terror.Error)"?

Reply:
It's OK to not change this code, just casually ask.

Reply:
yes and it's simple to change~